docs/packages/index.md

```diff
@@ -55,6 +55,17 @@ function updateDefaultHttpOptions({http}) {
 }
 ```
 
+## error
+Throws **HttpProviderError**. Its message property contains:
+
+```js
+{
+  result: 'Message or response body',
+  status: 200,
+  isAborted: false
+}
```

Comment:
Hmm I think it can also contain a header object. 
See https://github.com/cerebral/cerebral/blob/ftErrors/packages/cerebral-provider-http/src/DEFAULT_OPTIONS.js#L46